core/src/test/java/org/infinispan/statetransfer/RemoteGetDuringStateTransferTest.java

```diff
@@ -200,118 +200,6 @@ public void testScenario_111() throws Exception {
    }
 
    /**
-    * ISPN-3315: the remote get is trigger in stable state and the reply received after the rebalance_start command.
-    * However, the old owner will receive the request after the state transfer and he no longer has the key.
```

Comment:
I think you need to update the table at the top of the class as well. Actually I'd rather keep retrying without waiting for a new topology, just doing it from the `StateTransferInterceptor` instead of the `XDistributionInterceptor`.